modules/chapter1/pages/section4.adoc

```diff
@@ -0,0 +1,408 @@
+= KServe - Model Serving
+
+RHOAI has two main components for serving 
+
+* KServe Serving for single-model serving
+* KServe ModelMesh for multi-model serving
```

Comment:
With the introduction of single and multi-model serving options, we might need to reorganize sections a bit for clarity. The first 3 sections were written when only ModelMesh was an option.

Section 1 focuses on very high-level concepts and at the bottom introduces CRs for only Model Mesh. We might want to expand that part to explain that there are two flavors of model  serving (single and multi).

Section 2 is ok as is but we might want to rename it to something like `Multi-model serving with OpenVINO.

Section 3 is about custom model servers. I would move that section to be the last one.